<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>502</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>610</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>45</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 170, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>0</y>
         <width>241</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(85, 170, 255);</string>
       </property>
       <property name="text">
        <string>Controle de Vendas</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(15, 15, 15);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(35, 35, 35);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QTableWidget" name="tableWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>631</width>
            <height>391</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="gridStyle">
           <enum>Qt::NoPen</enum>
          </property>
          <column>
           <property name="text">
            <string>ID</string>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>50</weight>
             <bold>false</bold>
             <kerning>false</kerning>
            </font>
           </property>
           <property name="background">
            <color>
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Cliente</string>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Produto</string>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Quantidade</string>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Pre√ßo</string>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Pagamento</string>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
          </column>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_5">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(60, 60, 60);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QPushButton" name="btn_deletar">
          <property name="geometry">
           <rect>
            <x>470</x>
            <y>10</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton:pressed {
border-style: outset;
border-width:2px;
boder radius: 15px;
border color: black;
}</string>
          </property>
          <property name="text">
           <string>Deletar</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_baixar">
          <property name="geometry">
           <rect>
            <x>340</x>
            <y>10</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton: {
color: rgb(255, 255, 255);}

QPushButton:pressed {
border-style: outset;
border-width:2px;
boder radius: 15px;
border color: black;
}</string>
          </property>
          <property name="text">
           <string>Baixar</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_modificar">
          <property name="geometry">
           <rect>
            <x>200</x>
            <y>10</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton:pressed {
border-style: outset;
border-width:2px;
boder radius: 15px;
border color: black;
}</string>
          </property>
          <property name="text">
           <string>Modificar</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_adicionar">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>10</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton:pressed {
border-style: outset;
border-width:2px;
boder radius: 15px;
border color: black;
}</string>
          </property>
          <property name="text">
           <string>Adicionar</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>610</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
